app/views/includes/bulk_results_all.scala.html

```diff
@@ -25,49 +25,60 @@
 @failed = @{"failed"}
 
 <div class="divider--top clearfix">
-  <h3 id="all-calculations">@Messages("gmp.bulk.subheaders.allcalculations") (@bulkResultsSummary.total)</h3>
+    <hr class="govuk-section-break govuk-section-break--visible">
+
+    <h3 class="govuk-heading-s" id="all-calculations">@Messages("gmp.bulk.subheaders.allcalculations") (@bulkResultsSummary.total)</h3>
   <div class="column-three-quarters">
-    <ul class="bullets font-xsmall">
+    <ul class="govuk-list govuk-list--bullet font-xsmall">
       <li id="exp-all-1" class="font-xsmall">@Messages("gmp.bulk.explanations.all.1")</li>
       <li id="exp-all-2" class="font-xsmall">@Messages("gmp.bulk.explanations.all.2")</li>
     </ul>
   </div>
-  <div class="column-one-quarter text--right"><a id="download-all" data-journey-click="gmp-bulk:click:download_all" href="@routes.BulkResultsController.getResultsAsCsv(uploadReference, all)">
-  @Messages("gmp.download.link")<span class="visuallyhidden"> @Messages("gmp.download.description.all")</span></a></div>
+  <div class="column-one-quarter govuk-!-text-align-right"><a id="download-all" data-journey-click="gmp-bulk:click:download_all" href="@routes.BulkResultsController.getResultsAsCsv(uploadReference, all)">
+  @Messages("gmp.download.link")<span class="hidden"> @Messages("gmp.download.description.all")</span></a></div>
```

Comment:
Should the class be "govuk-visually-hidden"?

Reply:
This part as not done by me. SInce I branched off from your branch I think you are the right person to know :)

Reply:
I can change it if needed, please do confirm

Reply:
I have made a note, we will have the team discuss a final GMP clean up ticket that can be checked prior to final merge/release.